### ISSUE TITLE
[BUG] useSniffing parameter is actually dontUseSniffing

### ISSUE THREAD

**Issue by kammerjaeger** [elastic/ecs-dotnet#342]
**ECS integration/library project(s) (e.g. Elastic.CommonSchema.Serilog)**:
[Elastic.Serilog.Sinks](https://github.com/elastic/ecs-dotnet/tree/main/src/Elastic.Serilog.Sinks)

**ECS .NET assembly version (e.g. 1.4.2)**:
 8.6.1

**Description of the problem, including expected versus actual behavior**:
The parameter is the opposite of what its name is, that is quite confusing.
See: https://github.com/elastic/ecs-dotnet/blob/eac06c3df532d33c1889cca7dca388216c945dc4/src/Elastic.Serilog.Sinks/ElasticsearchSinkExtensions.cs#L81

Possible solution: https://github.com/elastic/ecs-dotnet/commit/0f0f89448a702c0103eabd5e4cb55df299b40e13

**Comment by Mpdreamz** [comment]
This took embarrassingly long to address, thank you for raising this @kammerjaeger. 

This should be fixed once #445 lands.

**Comment by kammerjaeger** [comment]
Maybe add a comment in the release note that this may break existing configurations when deployed

**Comment by Mpdreamz** [comment]
I changed the default to `false` so there is no breaking change, thanks for calling that out.

**Comment by kammerjaeger** [comment]
Good catch that is a problem, too. But I was more talking about the people like me who actively set it have to inverse the setting with the new release.
